### ISSUE TITLE
Uploaded image error

### ISSUE THREAD

**Issue by aerionTech** [artf/grapesjs#1462]
We have gotten GrapesJs integrated into our Angular application.  We have the asset manager showing a list of images that have been uploaded. Initially, there are no images and upon dragging an image and then handling the upload the asset manager shows the correct image. 

Upon uploading an image we are returning an object with one image in the data property, so the 1 image is appended to the existing. On initialisation, we are returning an object will the existing images. 

The problem exists in the uploaded image, it's shown in the editor with the "missing picture" image and does not show the uploaded image. The only way I can get the uploaded image to appear is to double click on

 the image and select it via the asset manager. 

I have attached a video showing the current issue and how it works. 

![ezgif-4-1f6ead2c18](https://user-images.githubusercontent.com/6082270/46115556-e0310200-c23a-11e8-9f0f-29e12bad3ec5.gif)

**Comment by aerionTech** [comment]
Hi That worked, in the documentation, you might want to update the documentation to remove the first line in the array and only use the object version for the return data. 

{
  data: [
    'https://.../image.png',
    // ...
    {
      src: 'https://.../image2.png',
      type: 'image',
      height: 100,
      width: 200,
    },
    // ...
  ]
}

**Comment by artf** [comment]
@aerionTech right, but I think I'll update the upload function to make it understand also string items

**Comment by aerionTech** [comment]
Good idea, more flexibility the better, especially if you don't wan to specify a size for the image and just provide the url for it.
